osu.Game/Tests/Visual/RealtimeMultiplayer/TestRealtimeMultiplayerClient.cs

```diff
@@ -31,7 +31,7 @@ public class TestRealtimeMultiplayerClient : StatefulMultiplayerClient
         public void RemoveUser(User user)
         {
             Debug.Assert(Room != null);
-            ((IMultiplayerClient)this).UserLeft(Room.Users.Single(u => u.User == user));
+            ((IMultiplayerClient)this).UserLeft(new MultiplayerRoomUser(user.Id));
```

Comment:
it this change required? seems like the previous logic is more correct (obviously this is a test so it doesn't matter so much, but this will potentially be providing unfilled details and break reference comparisons).

Reply:
It is for the removal test to work. The model implements `IEquatable` and I believe that is what's going to be used?

Reply:
aha. guess it's ok